lib/kwtool/crawler/upload_parser.ex

```diff
@@ -15,5 +15,11 @@ defmodule Kwtool.Crawler.UploadParser do
     end
   end
 
-  def parse(_), do: {:error, :file_is_invalid}
+  def parse!(_), do: raise("Uploaded keyword file is invalid")
+
+  def parse_asset_type(%Plug.Upload{content_type: "csv" <> _}), do: {:ok, :csv}
+  def parse_asset_type(_), do: {:error, :unsupported_asset_type}
+
+  def supported_mime_types, do: ~w(text/csv)
+  def max_file_size_in_bytes, do: 3_000_000
```

Comment:
Should we move these two to `KwtoolWeb.Api.V1.UploadParams`? as this logic related to the params validation